gNMI_Interface/04__Subscribe.robot

```diff
@@ -3,92 +3,97 @@ Documentation    Generic device agnostic test suite for gNMI ``Subscribe`` RPC/o
 Test Tags        subscribe
 
 Resource         Subscribe.resource
+Library          SubscribeLibrary.py  ${LIB_CONFIG}
 
 Resource         gNMIClient.resource
-Suite Setup      Setup gNMI Client
-Suite Teardown   Close gNMI Client
+Test Setup       Setup gNMI Client
+Test Teardown    Close client
 
 
 *** Test Cases ***
 
 Basic subscription with "mode" parameter
     [Tags]    sanity
-    [Documentation]  Test that the device correctly responds to all three
-    ...    "Subscribe" request modes.  No further functionality (such as actually
+    [Documentation]    Test that the device correctly responds to all three
+    ...    "Subscribe" request modes.    No further functionality (such as actually
     ...    polling the device) is tested.
-    [Template]    Verify Subscribe
+    [Template]    Subscribe ${mode} to default path with default encoding
     STREAM
     ONCE
     POLL
 
-Subscribe to non-existent "prefix"
-
-Subscribe to non-existent "path"
-
-# message Subscription {
-#   Path path = 1;                    // The data tree path.
-#   SubscriptionMode mode = 2;        // Subscription mode to be used.
-#   uint64 sample_interval = 3;       // ns between samples in SAMPLE mode.
-#   bool suppress_redundant = 4;
-#   uint64 heartbeat_interval = 5;
-# }
-
 Subscribe ONCE with supported "encoding" values
-
-Two Subscriptions in single request - with same "path"
-
-Two Subscriptions in single request - with different "path"
-
-QOS marked subscriptions
-
-STREAM with TARGET_DEFINED mode
+    [Tags]    sanity
+    [Documentation]    Verify that the device is able to respond correctly for
+    ...    all declared encodings.
+    Given device capabilities
+    and subscription paths    ${GET-PATH}
+    Then subscribe ONCE with supported encodings
+
+Two subscriptions in single request with the same "path"
+    [Documentation]    Verify that the device can handle ONCE subscription
+    ...    with two identical paths.
+    Given subscription paths    ${GET-PATH}    ${GET-PATH}
+    And subscription ONCE with default encoding
+    Then device responds
+
+Two subscriptions in single request with different "path"
+    [Documentation]    Verify that the device can handle ONCE subscription
+    ...    with two different paths.
+    Given subscription paths    ${GET-PATH}    ${SECONDARY-PATH}
+    And subscription ONCE with default encoding
+    Then device responds
+
+Subscribe ONCE sends final message with "sync_response"
+    [Documentation]    When a ONCE subscription is created, the device must
+    ...    respond with a series of responses terminated by an empty
+    ...    response with "sync_response".
+    Given Subscription paths    ${GET-PATH}
```

Comment:
Why now  capital letter `Subscription paths`?  In previous test it was lowercase.